web/src/app/util/AppLink.tsx

```diff
@@ -1,15 +1,17 @@
 import React, { forwardRef, ForwardRefRenderFunction } from 'react'
 import { Link, LinkProps, useLocation } from 'react-router-dom'
 import joinURL from './joinURL'
+import { OpenInNew } from '@mui/icons-material'
 
 export interface AppLinkProps extends LinkProps {
   to: string
   newTab?: boolean
+  icon?: boolean
```

Comment:
The idea was to have the icon be in a single element for when the mouse pointer changes to a hand. We can probably do one of these but I feel like it would be an API enhancement overall.

Reply:
I disagree, but sure.

Reply:
Pushed up a solution using CSS.